layouts/default.vue

```diff
@@ -117,7 +117,8 @@ export default {
     ]),
     rasterLayer () {
       const rasterLayer = getRasterLayer()
-      rasterLayer.source.tiles = _.get(this.activeRasterData, 'tiles')
+      rasterLayer.source.tiles = [_.get(this.activeRasterData, 'url')]
```

Comment:
What happens if activeRasterData is not set.

Reply:
Then no layer is displayed. This is caught in the raster layer file.